From b389046aba0486a834b18021a5de6af12919a265 Mon Sep 17 00:00:00 2001
From: Erich Hauptli <ejhauptl@us.ibm.com>
Date: Mon, 8 May 2017 16:08:13 -0500
Subject: [PATCH 3/3] Adjusting A Bus Frequency

Per Joe McGill, lowering the A bus frequency to support the slower 20Gbs
OBUS rate.
---
 witherspoon.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/witherspoon.xml b/witherspoon.xml
index ed4f21e..3477f03 100644
--- a/witherspoon.xml
+++ b/witherspoon.xml
@@ -9727,7 +9727,7 @@
 	</attribute>
 	<attribute>
 		<id>FREQ_A_MHZ</id>
-		<default>0x1900</default>
+		<default>0x1250</default>
 	</attribute>
 	<attribute>
 		<id>FREQ_CORE_MAX</id>
-- 
1.8.2.2

